.github/workflows/ansible-test.yml

```diff
@@ -1,93 +1,176 @@
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
         with:
-          # FIXME ansible_collections/NAMESPACE/COLLECTION
-          path: ansible_collections/community/FIXME
+          path: ansible_collections/NAMESPACE/COLLECTION_NAME
 
-      - name: Set up Python 3.6
-        uses: actions/setup-python@v1
+      - name: Set up Python ${{ matrix.ansible }}
+        uses: actions/setup-python@v2
         with:
-          python-version: 3.6
+          python-version: ${{ matrix.python }}
 
+      # Install the head of the given branch (devel, stable-2.10)
       - name: Install ansible-base (${{ matrix.ansible }})
         run: pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible }}.tar.gz --disable-pip-version-check
 
+      # run ansible-test sanity inside of Docker.
+      # The docker container has all the pinned dependencies that are required.
+      # Explicity specify the version of Python we want to test
       - name: Run sanity tests
-        run: ansible-test sanity --docker -v --color --python 3.6
+        run: ansible-test sanity --docker -v --color --python ${{ matrix.python }}
+        working-directory: ./ansible_collections/NAMESPACE/COLLECTION_NAME
+
+###
+# Unit tests (OPTIONAL)
+#
+# https://docs.ansible.com/ansible/latest/dev_guide/testing_units.html
 
-# If you don't have unit tests, you can delete this section
   units:
     runs-on: ubuntu-latest
+    name: Units (Ⓐ${{ matrix.ansible }}+py${{ matrix.python }}})
+    strategy:
+      fail-fast: false
+      matrix:
+        ansible:
+          # - stable-2.9 # Only if your collection supports Ansible 2.9
+          - stable-2.10
+          - devel
+        python:
+          - 2.7
+          - 3.7
+        exclude:
+          - python: 3.8  # blocked by ansible/ansible#70155
+
     steps:
       - name: Check out code
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
         with:
-          # FIXME ansible_collections/NAMESPACE/COLLECTION        
-          path: ansible_collections/community/grafana
+          path: ansible_collections/NAMESPACE/COLLECTION_NAME
 
-      - name: Set up Python 3.6
-        uses: actions/setup-python@v1
+      - name: Set up Python ${{ matrix.ansible }}
+        uses: actions/setup-python@v2
         with:
-          python-version: 3.6
+          python-version: ${{ matrix.python }}
+
+      - name: Install ansible-base (${{ matrix.ansible }})
+        run: pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible }}.tar.gz --disable-pip-version-check
 
-      - name: Install ansible-base (devel)
-        run: pip install https://github.com/ansible/ansible/archive/devel.tar.gz --disable-pip-version-check
+      # OPTIONAL If your unit test requires Python libraries from other collections
+      # Install them like this
+      - name: Install collection dependencies
+        run: ansible-galaxy collection install ansible.netcommon -p .
```

Comment:
So what I'm proposing is this:
```suggestion
        run: ansible-galaxy install -r test-requirements.yml -p .
```
And have `ansible.netcommon` added to that location instead of vendor-locking on GHA. This would allow to install the same set of test deps on any number of different CIs/envs as well as locally:
```yaml
# test-requirements.yml
---
collections:
- name: ansible.netcommon
```